chains/btc/executor/executor.go

```diff
@@ -285,8 +286,19 @@ func (e *Executor) outputs(tx *wire.MsgTx, proposals []*BtcTransferProposal) (ui
 		if err != nil {
 			return 0, err
 		}
+
+		opReturnData := []byte(strconv.Itoa(int(prop.Source)) + "_" + strconv.Itoa(int(prop.Data.DepositNonce)))
```

Comment:
We should at least add output index to this so it easy to differentitate which output is targeted with multiple proposals.